presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1536,6 +1577,8 @@ public HiveInsertTableHandle beginInsert(ConnectorSession session, ConnectorTabl
                 .collect(toList());
 
         HiveStorageFormat tableStorageFormat = extractHiveStorageFormat(table.get());
+
+        // TODO (mayankgarg): Encryption support needs to come here as well
```

Comment:
you've got a todo left over :)

Reply:
This is for the write part - added this while I was trying to understand the code. Will remove this.